com.github.elth0r0.iqpuzzle.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.elth0r0.iqpuzzle
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: iqpuzzle
+separate-locales: false
+rename-icon: iqpuzzle
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
you don't need this and fallback-x11. just one or the other (fallback-x11 means wayland will be chosen by default if available)

Reply:
You should have instead removed socket=x11 and not fallback-x11.

Reply:
Many thanks for the information! The explanation in the Flatpak docs wasn't 100% clear to me, which combination should be used. Now I exchanged x11 with fallback-x11.